### PR TITLE
Deserialize CreationDate for TaintRaised

### PR DESCRIPTION
**Pull request by georgii-borovinskikh-sonarsource**
Closes https://github.com/SonarSource/sonarlint-visualstudio/issues/3706

### CODE REVIEW

File: SonarQube.Client/Helpers/MillisecondUnixTimestampDateTimeOffsetConverter.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
```
Comment (rita-sonarsource) [inline SonarQube.Client/Helpers/MillisecondUnixTimestampDateTimeOffsetConverter.cs:1]: copyright

File: SonarQube.Client/Helpers/MillisecondUnixTimestampDateTimeOffsetConverter.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+using Newtonsoft.Json;
+
+namespace SonarQube.Client.Helpers
+{
+    public class MillisecondUnixTimestampDateTimeOffsetConverter : JsonConverter<DateTimeOffset>
```
Comment (rita-sonarsource) [inline SonarQube.Client/Helpers/MillisecondUnixTimestampDateTimeOffsetConverter.cs:6]: Since this is an infra module, add tests for this class

**Review by rita-sonarsource — CHANGES_REQUESTED** [review]
LGTM see comments

**Review by rita-sonarsource — APPROVED** [review]

**Comment by duncanp-sonar** [comment]
Is the format of this date-time different from those returned in SonarQube/SonarCloud APIs?
How do we handle deserializing those other timestamps?

**Pull request merged**